docs/guide/execute-transaction.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: splash
+---
+
+# Execute Transaction
+
+`executeTransaction` function can be used to perform many transactions. It supports every transaction which is possible in algorand network. Ex: Deploy ASA/SSC, Opt-In, Transfers, Delete, Destroy etc. `executeTransaction` takes `ExecParams` or `ExecParams[]` as parameter.
+If you pass an array of `ExecParams`, it will be considered as `atomic transaction`.
+In below sections we will demonstrate how to pass these parameters.
+
+Note: For each parameter `type` and `sign` fields are mandatory.
+  - `type`: type of transaction
+  - `sign`: sign type
+
+Examples of parameter(`ExecParams`):
+# Transfer Algo using secret key
+
+```js
+  {
+    type: TransactionType.TransferAlgo,
+    sign: SignType.SecretKey,
+    fromAccount: john,
+    toAccountAddr: alice.address,
+    amountMicroAlgos: 100,
+    payFlags: { totalFee: 1000 }
+  }
+```
+
+# Transfer Algo using logic signature
+
+```js
+  {
+    type: TransactionType.TransferAlgo,
+    sign: SignType.LogicSignature,
+    fromAccountAddr: contractAddress,
+    toAccountAddr: alice.address,
+    amountMicroAlgos: 100,
+    lsig: lsig,
+    payFlags: { totalFee: 1000 }
+  }
+```
+
+# Deploy ASA
+
+```js
+  {
+    type: TransactionType.DeployASA,
+    sign: SignType.SecretKey,
+    fromAccount: john,
+    asaName: 'gold',
+    payFlags: { totalFee: 1000 }
+  }
+```
+
+# Opt-In to ASA
+
+```js
+  {
+    type: TransactionType.OptInASA,
+    sign: SignType.SecretKey,
+    fromAccount: alice,
+    assetID: assetIndex,
+    payFlags: { totalFee: 1000 }
+  }
+```
```

Comment:
Let's cover both cases: `optInLsigToASA`, `optInAccountToASA`. Same for SSC.

Reply:
https://github.com/scale-it/algo-builder/blob/master/docs/guide/deployer.md#optin-to-asa , these are already covered, the main goal here is to `how to pass params`

Reply:
Please add this link here, remember these are user docs

Reply:
user will start from `deployer`, that's the essential part, then after reading that user will come here which is more advanced than deployer. adding executeTransaction link in deployer is better